base/kra/shared/webapps/kra/WEB-INF/web.xml

```diff
@@ -32,7 +34,7 @@
     <security-constraint>
         <web-resource-collection>
             <web-resource-name>Agent Services</web-resource-name>
-            <url-pattern>/rest/agent/*</url-pattern>
+            <url-pattern>/v1/agent/*</url-pattern>
```

Comment:
Do we need to keep the old URL?

Reply:
> Do we need to keep the old URL?

You are right, it is not needed. Removed!